lib/screens/admin_new_product_page.dart

```diff
@@ -493,6 +494,72 @@ class _AdminNewProductPageState extends State<AdminNewProductPage> {
     );
   }
 
+  Widget _buildDeleteStockItemButton(Stock stock) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: <Widget>[
+        Container(
+          alignment: AlignmentDirectional.center,
+          height: 20,
```

Comment:
Since we're setting the icon.size as 20 at L509, do we still need to set the size limits for this container?

Reply:
The size-limit for this container is for padding the IconButton.
I just adjusted the size-limit and then the 'delete icon' becomes symmetrical.